guides/source/getting_started.md

```diff
@@ -388,9 +388,9 @@ create and read. The form for doing this will look like this:
 ![The new article form](images/getting_started/new_article.png)
 
 It will look a little basic for now, but that's ok. We'll look at improving the
-styling for it afterwards.
+styling for it afterward.
 
-### Laying down the ground work
+### Laying down the groundwork
```

Comment:
I am unable to squash the commits to one, as I made a major merge in between. So, I have created a new PR for the same change.
Closing this PR.
The new PR is: https://github.com/rails/rails/pull/24942